sql/core/src/test/resources/sql-tests/inputs/aliased-subquery-in-from.sql

```diff
@@ -0,0 +1,14 @@
+-- Aliased subqueries in FROM clause
```

Comment:
Could you move the test case `sql-tests/inputs/aliased-subquery-in-from.sql` to `sql-tests/inputs/subquery/subquery-in-from.sql`?